sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2995,6 +3260,26 @@ def hpat_pandas_series_nunique_impl(self, dropna=True):
 @overload_method(SeriesType, 'count')
 def hpat_pandas_series_count(self, level=None):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.count
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/count.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the number of not NaN values in Series
+       :name: ex_series_count
+
+    .. code-block:: console
+
+        > python ./series/count.py
+        2
+
+    Intel Scalable Dataframe Compiler Developer Guide
```

Comment:
Missing See Also

Reply:
Pandas API does not have See Also section for count

Reply:
Ok. I am putting myself in user shoulders. I know there is count() and value_count() functions, and shame on me, I constantly forget about their differences. Do you think I am alone in that? So it will be a bit user friendly for those who expressed interest in learning about count() functions to let them know all other API variants that exist for counting elements in Pandas. I would include methods like len(), count(), value_count(), and all other methods, referring to each other in from respective See Also sections. Does it make sense?

Reply:
Yes, you are right. I will improve it